CONTRIBUTING.md

```diff
@@ -35,49 +51,23 @@ add appropriate metadata before the issue is addressed.
-or if it should land then it should be escalated to the WG. WG members should regularly
-discuss pending contributions in order to find a resolution. It is expected that only a
-small minority of issues be brought to the WG for resolution and that discussion and
-compromise among collaborators be the default resolution mechanism.
+For non-trivial contributions, pull requests should sit for at least 36 hours to ensure that contributors in other timezones have time to review. Consideration should also be given to weekends and other holiday periods to ensure active collaborators all have reasonable time to become involved in the discussion and review process if they wish.
 
-# Becoming a Member and Collaborator
+The default for each contribution is that it is accepted once no collaborator has an objection. During review collaborators may also request that a specific contributor who is most versed in a particular area gives a "LGTM" before the PR can be merged. There is no additional "sign off" process for contributions to land. Once all issues brought by collaborators are addressed, it can be landed by any collaborator.
 
-To become a member of the Community Committee, start participating! Attend the bi-weekly meetings, investigate issues tagged as `good first issue`,
-file issues and pull requests, and provide insight via GitHub. Request to become an Observer by
-filing an issue. Once added as an Observer to meetings, we will track attendance and participation
-for 3 months, as according to our [governance guidelines](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md#section-4-establishment-of-the-community-committee). If you meet the minimum
-attendance and are participating, the CommComm will vote to add you as a member.
+In the case of an objection being raised in a pull request by another collaborator, all involved collaborators should seek to arrive at a consensus by way of addressing concerns being expressed by discussion, compromise on the proposed change, or withdrawal of the proposed change.
 
-All participants added as members should be on-boarded in a timely manner, added as a collaborator, and be given write access to the repository.
+If a contribution is controversial and collaborators cannot agree about how to get it to land or if it should land then it should be escalated to the WG. WG members should regularly discuss pending contributions in order to find a resolution. It is expected that only a small minority of issues be brought to the WG for resolution and that discussion and compromise among collaborators be the default resolution mechanism.
 
-Collaborators are expected to follow this policy and continue to send pull requests, go through
-proper review, and have other collaborators merge their pull requests.
+Collaborators are expected to follow this policy and continue to send pull requests, go through proper review, and have other collaborators merge their pull requests.
```

Comment:
I liked @bnb suggestion about mentioning that the contributions should be constructive. I also think we should explicitly say that the contribution type depends on the WG and that it's not necessarily code.

Reply:
+1, though I don't think this should be a blocker for this PR. We can and should definitely follow up with more tweaks to `CONTRIBUTING.md` as a living document rather than a de facto and unchangeable source of truth 😄